### PR TITLE
Poll about switching to GitHub discussions

### PR DESCRIPTION
**Pull request by juliogonzalez**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -234,20 +234,20 @@ <h3>Latest news</h3>
           <!-- News box Start -->
           <div class="news-box">
             <div class="news-box-title">
-              <span>21/03/2023</span>
+              <span>05/04/2023</span>
             </div>
             <div class="news-box-content">
-              <p>A Security fix is now released to be installed on top of Uyuni 2023.03. Check the <a href="https://lists.opensuse.org/archives/list/announce@lists.uyuni-project.org/thread/3CWLDXN3YJ7OWQPPIGOQUQFNEE3PUIZR/" target="_blank">announcement</a> to get all the details about the fix and instructions to install the patch. We strongly recommend you update as soon as possible.</p>
+              <p>We started a <a href="https://github.com/uyuni-project/uyuni/discussions/6821" target="_blankl">poll</a> to decide if we to switch async discussions from <a href="../pages/contact.html#ml">Mailing lists</a> to <a href="https://github.com/uyuni-project/uyuni/discussions" target="_blank">GitHub conversations</a>.</p>
```
Comment (deneb-alpha) [inline index.html:240]:
```suggestion
              <p>We started a <a href="https://github.com/uyuni-project/uyuni/discussions/6821" target="_blankl">poll</a> to decide if we want to switch async discussions from <a href="../pages/contact.html#ml">mailing lists</a> to <a href="https://github.com/uyuni-project/uyuni/discussions" target="_blank">GitHub conversations</a>.</p>
```

File: pages/news.html [hunk 2]
```diff
@@ -88,6 +88,7 @@ <h1>NEWS</h1>
 <div class="container page-wrapper">
     <div class="container">
         <div class="row content-padding subpage-content">
+            <p><strong>05/04/2023</strong> - We started a <a href="https://github.com/uyuni-project/uyuni/discussions/6821" target="_blankl">poll</a> to decide if we to switch async discussions from <a href="../pages/contact.html#ml">Mailing lists</a> to <a href="https://github.com/uyuni-project/uyuni/discussions" target="_blank">GitHub conversations</a>.</p>
```
Comment (deneb-alpha) [inline pages/news.html:91]:
```suggestion
            <p><strong>05/04/2023</strong> - We started a <a href="https://github.com/uyuni-project/uyuni/discussions/6821" target="_blankl">poll</a> to decide if we want to switch async discussions from <a href="../pages/contact.html#ml">mailing lists</a> to <a href="https://github.com/uyuni-project/uyuni/discussions" target="_blank">GitHub conversations</a>.</p>
```

**Review by deneb-alpha — CHANGES_REQUESTED** [review]
two small changes  with suggestions :)

File: pages/news.html (hunk 2 above)
Comment (juliogonzalez) [reply pages/news.html:91]: In fact I wanted to avoid "want to". So will remove that.

File: index.html [hunk 3]
```diff
@@ -234,20 +234,20 @@ <h3>Latest news</h3>
           <!-- News box Start -->
           <div class="news-box">
             <div class="news-box-title">
-              <span>21/03/2023</span>
+              <span>05/04/2023</span>
             </div>
             <div class="news-box-content">
-              <p>A Security fix is now released to be installed on top of Uyuni 2023.03. Check the <a href="https://lists.opensuse.org/archives/list/announce@lists.uyuni-project.org/thread/3CWLDXN3YJ7OWQPPIGOQUQFNEE3PUIZR/" target="_blank">announcement</a> to get all the details about the fix and instructions to install the patch. We strongly recommend you update as soon as possible.</p>
+              <p>We started a <a href="https://github.com/uyuni-project/uyuni/discussions/6821" target="_blankl">poll</a> to decide if we want to switch async discussions from <a href="../pages/contact.html#ml">mailing lists</a> to <a href="https://github.com/uyuni-project/uyuni/discussions" target="_blank">GitHub conversations</a>.</p>
```
Comment (juliogonzalez) [inline index.html:240]:
```suggestion
              <p>We started a <a href="https://github.com/uyuni-project/uyuni/discussions/6821" target="_blankl">poll</a> to decide if we switch async discussions from <a href="../pages/contact.html#ml">mailing lists</a> to <a href="https://github.com/uyuni-project/uyuni/discussions" target="_blank">GitHub conversations</a>.</p>
```

**Review by deneb-alpha — APPROVED** [review]
LGTM :) 
thanks!

**Pull request merged**